onnxruntime/core/mlas/lib/reorder.cpp

```diff
@@ -396,103 +411,180 @@ Return Value:
 }
 
 void
-MLASCALL
-MlasReorderOutputNchw(
-    const int64_t* OutputShape,
-    const float* S,
-    float* D
+MlasReorderOutputNchwThreaded(
+    void* Context,
```

Comment:
>void [](http://example.com/codeflow?start=4&length=4)

I would suggest to pass the actual type.

Reply:
If you are referring to the type of `Context`, the `MlasReorderOutputNchwThreaded` function is called from `MlasExecuteThreaded`, which in turn gets (and passes) `Context` as an argument of type `void *`. Am I missing something?